internal/config/config.go

```diff
@@ -43,33 +44,41 @@ func (t *Test) Validate() (err error) {
 	return nil
 }
 
+// IncreaseOnError is a function that reverses a test when an error occurs.
 func (t *Test) IncreaseOnError() (increase int) {
 	if t.Reverse {
 		return 0
 	}
 	return 1
 }
 
+// IncreaseOnSuccess is a function that reverses the test when there is an succes
```

Comment:
IncreaseOnSuccess is a function that reverses the test result adder when there is an succes